pkgs/servers/sql/cockroachdb/default.nix

```diff
@@ -13,13 +13,13 @@ let
 in
 buildGoPackage rec {
   pname = "cockroach";
-  version = "19.1.1";
+  version = "19.1.4";
 
   goPackagePath = "github.com/cockroachdb/cockroach";
 
   src = fetchurl {
     url = "https://binaries.cockroachdb.com/cockroach-v${version}.src.tgz";
-    sha256 = "1vbz5j0y9ri8c99k8fc5rnwigay478p1mac5g402639ilkqd41fc";
+    sha256 = "d819167dc109b566511cb6cef9aadc8c4f07b1be6a8b3108f4bbd14808d21faf";
```

Comment:
This does not look right - try this one https://github.com/rvolosatovs/nixpkgs/commit/8c50f0952d4fd8eab11f304b8de888c6959523c1#diff-538ae77f6d84d5f7800d94b0d433dd41R22 ?

Reply:
Done. Thanks for correcting me! How did you obtain that hash?

Reply:
The *correct* way:
```
nix-prefetch-url https://binaries.cockroachdb.com/cockroach-v19.1.4.src.tgz
[98.3 MiB DL]
path is '/nix/store/jl9a12xqv3dxb87c5pwjy422gyfap42f-cockroach-v19.1.4.src.tgz'
1bqzs844ildvyh4332vapsqhfkwcvjmgkkmn3i8ndd89q5yic6fq
```
Note, that there are also `nix-prefetch-git`, `nix-prefetch-hg` etc. available

However, I'm lazy and most of the time I just modify the hash to something random(but valid), let the build fail and use the hash from the build. It looks like something like this(the hash is for a different package):
```
hash mismatch in fixed-output derivation '/nix/store/91qiinmi5302rrxa1q6qydpzynn8m7l8-tools-6b3d1c9':
  wanted: sha256:16cfzmfr9jv8wz0whl433xdm614dk63fzjxv6l1xvkagjmki49iy
  got:    sha256:0a2xjx9hqkash7fd2qv9hd93wcqdbfrmsdzjd91dwvnk48j61daf
```
But I would not recommend this approach, since it's extremely error-prone and you can very easily run into issues with e.g. caching